### ISSUE TITLE
conflicting inspections uniqid()

### ISSUE THREAD

**Issue by Hikariii** [kalessil/phpinspectionsea#1269]
| Subject        | Details                                                                       |
| :------------- | :---------------------------------------------------------------------------- |
| Plugin         | Php Inspections (EA Extended) |
| Language level | PHP 7.3                        |

Current behaviour
--
When using uniqid() without the `more entropy` parameter it give a warning.
The inspection wants the method to be called explixitly with true/false. (correct in my opinion).
![afbeelding](https://user-images.githubusercontent.com/1484903/55382498-13c05200-5526-11e9-938b-08842798bbc8.png)

But when I explicitly use `false` as the `more entropy` argument, the inspection tells me to get rid of the last parameter as it is the default value that I give as an argument.
![afbeelding](https://user-images.githubusercontent.com/1484903/55382710-a5c85a80-5526-11e9-9127-f72f9b8ab454.png)

code snippet:
```php
<?php

$userAssetsToken = uniqid('TEST', false);
```

Expected behaviour
--
I expect the inspection to not give a notice that an argument can be dropped when a different inspection requires me to add it.
Now there is no valid way to use `uniqid('PREFIX', false)`.

Environment details
---
![afbeelding](https://user-images.githubusercontent.com/1484903/55382876-140d1d00-5527-11e9-9816-3b32d9fce0a8.png)

**Comment by kalessil** [comment]
Fixed!
